### ISSUE TITLE
.DS_Store files cause exception

### ISSUE THREAD

**Issue by t-unit** [jyotirmoy-paul/cached_build_runner#4]
Running `flutter packages pub run cached_build_runner build` fails for when encountering a `.DS_Store` file:

```
Unhandled exception:
FileSystemException: Failed to decode data using encoding 'utf-8', path = '/.../test/.DS_Store'
#0      _File._tryDecode (dart:io/file_impl.dart:558:7)
#1      _File.readAsStringSync (dart:io/file_impl.dart:578:7)
#2      CachedBuildRunner._fetchFilePathsFromTest (package:cached_build_runner/cached_build_runner.dart:312:36)
#3      CachedBuildRunner.build (package:cached_build_runner/cached_build_runner.dart:146:29)
#4      BuildCommand.run (package:cached_build_runner/commands/build_command.dart:30:30)
<asynchronous suspension>
#5      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
pub finished with exit code 255
make[1]: *** [generate-code] Error 255
make: *** [setup] Error 2
````

Most likely those files should just get ignored.

**Comment by jyotirmoy-paul** [comment]
Got it, will look into this. For now I guess you can just remove the `.DS_Store` file and try.

**Comment by jyotirmoy-paul** [comment]
@t-unit Please let me know if the issue gets fixed in the latest version `0.0.6`

**Comment by t-unit** [comment]
@jyotirmoy-paul thx, the problem is solved
